components/trustpilot/sources/new-service-reviews/new-service-reviews.mjs

```diff
@@ -1,40 +1,101 @@
-import {
-  SORT_OPTIONS,
-  SOURCE_TYPES,
-} from "../../common/constants.mjs";
-import common from "../common/polling.mjs";
+import trustpilot from "../../trustpilot.app.mjs";
 
 export default {
-  ...common,
   key: "trustpilot-new-service-reviews",
   name: "New Service Reviews",
-  description: "Emit new event when a customer posts a new service review on Trustpilot. This source periodically polls the Trustpilot API to detect new service reviews, combining both public and private reviews for comprehensive coverage. Each event contains the complete review data including star rating, review text, consumer details, business unit info, and timestamps. Ideal for monitoring overall business reputation, tracking customer satisfaction metrics, and triggering workflows based on review ratings or content.",
-  version: "0.0.3",
+  description: "Emit new event when a customer posts a new service review on Trustpilot. This source periodically polls the Trustpilot API to detect new service reviews using the private reviews API for comprehensive coverage. Each event contains the complete review data including star rating, review text, consumer details, business unit info, customer email, and timestamps. Ideal for monitoring overall business reputation, tracking customer satisfaction metrics, and triggering workflows based on review ratings or content.",
+  version: "0.1.0",
   type: "source",
   dedupe: "unique",
-  methods: {
-    ...common.methods,
-    getSourceType() {
-      return SOURCE_TYPES.NEW_REVIEWS;
+  props: {
+    trustpilot,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 15 * 60, // 15 minutes
```

Comment:
I don't see why the common file should be removed, there is plenty of code being duplicated across the sources which should remain in that file instead